WooCommerce/Classes/ViewRelated/Products/Edit Product/EditCategories/ProductCategoryListViewModel.swift

```diff
@@ -129,6 +110,13 @@ private extension ProductCategoryListViewModel {
             DDLogError("⛔️ Error fetching product categories: \(rawError.localizedDescription)")
         }
     }
+
+    /// Updates  `categoryViewModels` from  the resultController's fetched objects.
+    ///
+    func updateViewModelsArray() {
+        let fetchedCategories = resultController.fetchedObjects
+        categoryViewModels = ProductCategoryViewModelBuilder.viewModels(from: fetchedCategories, selectedCategories: product.categories)
```

Comment:
Do you think we should run the `viewModels` calculation in the background? Or maybe only do that once we run into performance issues?

Reply:
I was thinking about that yesterday and decided to not pre optimize at this point. 🙈 
I could add a note on the main issue just to be aware of it towards the end of the feature.

Reply:
Yeah, it's definitely pre-op. 

What do you think about adding a [performance test](https://developer.apple.com/documentation/xctest/xctestcase/1496290-measure) in the unit tests for now?